nixos/lib/testing/meta.nix

```diff
@@ -22,7 +22,7 @@ in
           };
           timeout = lib.mkOption {
             type = types.nullOr types.int;
-            default = 3600;  # 1 hour
+            default = null; # NOTE: null values are filtered out by `meta`.
```

Comment:
Looks unrelated. Worth submitting separately?

Reply:
This should be temporary for the jobset, there's a Hydra bug...

Reply:
It's not, unfortunately. The issue is that the 1 hour timeout for tests applies to the entire test closure, which is not a problem for big jobsets that build everything (including the test dependencies) as separate jobs, but becomes a problem for small jobsets like this one.

Reply:
So I guess we have to keep this in for the small jobset, then back out when we upgrade it to a full sized one.